src/css/structure/modules/infragistics.ui.tree.css

```diff
@@ -39,7 +39,7 @@
 	height:16px;
 }
 .ui-igtree a { 
-    text-decoration: none;
+    text-decoration: none!important;
```

Comment:
Shouldn't this be applied on `.ui-igtree a.ui-state-hover`? I believe the `!important` on the general anchor is not the best solution here.